src/styles/blocks/card.scss

```
@@ -0,0 +1,123 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
```

You can put a space here.

```suggestion
  flex-wrap: wrap;
  
  box-sizing: border-box;
```